src/hooks/useColorMode.ts

```diff
@@ -14,8 +14,8 @@ const useColorMode: UseColorMode = () => {
 		if (colorMode) {
 			setTheme(colorMode)
 			document.documentElement.dataset.theme = colorMode
-			// @ts-ignore
-			window?.__setTheme?.(colorMode)
+			// @ts-ignore __setTheme global has no type yet
```

Comment:
sometimes it makes sense to use ts-ignore but I like to comment on `why` it's being used